Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -2839,6 +2839,7 @@ void WebProcessProxy::wrapCryptoKey(Vector<uint8_t>&& key, CompletionHandler<voi
 void WebProcessProxy::serializeAndWrapCryptoKey(WebCore::CryptoKeyData&& keyData, CompletionHandler<void(std::optional<Vector<uint8_t>>&&)>&& completionHandler)
 {
     auto key = WebCore::CryptoKey::create(WTFMove(keyData));
+    MESSAGE_CHECK(key->algorithmIdentifier() != CryptoAlgorithmIdentifier::DEPRECATED_SHA_224);
     MESSAGE_CHECK_COMPLETION(key, completionHandler(std::nullopt));
```

Comment:
This needs to go below `MESSAGE_CHECK_COMPLETION(key, completionHandler(std::nullopt));` as if the key is null 💥